codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -691,6 +693,7 @@ getPictureCS (Sector cs _ _ _) = cs
 getPictureCS (Arc cs _ _ _) = cs
 getPictureCS (ThickArc cs _ _ _ _) = cs
 getPictureCS (Text cs _) = cs
+getPictureCS (Blank _) = emptyCallStack
```

Comment:
You have a call stack here.  I'm not sure why the original code doesn't have a CallStack for `Pictures`, but I think you should go ahead and produce the one you've got.

Reply:
Were you going to return the stored call stack here?

Reply:
sorry. I had initially compared it to Pictures, so was using emptyCallStack to get it to type check :/